velox/functions/prestosql/Map.cpp

```diff
@@ -213,6 +296,35 @@ class MapFunction : public exec::VectorFunction {
     }
     return true;
   }
+
+  void checkDuplicateConstantKeys(
```

Comment:
A generic function to check for duplicate in a vector range sounds like something pretty useful. Should we consider moving this to base vector?

Reply:
Sure. I'll move it.

Reply:
Added: #4892